.gitignore

```diff
@@ -1,3 +1,7 @@
+# SPDX-FileCopyrightText: 2012 meta-raspberrypi contributors
+#
```

Comment:
I see that there is a difference between the XML comment and shell one (there is an additional newline between the SPDX headers). Is there a standard in that regard? Can we use it to have a uniform format?